src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -47,6 +48,12 @@ static const int32_t trigger_matches[] = {
 	SR_TRIGGER_ONE,
 };
 
+static const char* external_clock_edges[] = {
+	"r", // positive edge
+	"f"  // negative edge
```

Comment:
I think we really should have "rising" and "falling" as strings here. "r" and "f" as options look cryptic in Pulseview.

Reply:
OK, sure. No problem. Changed.